### PR TITLE
Adding itch.io extension

### PR DESCRIPTION
**Pull request by Aayush9029**
## Description

Browse through 1000000+ games from [itch.io](https://itch.io)

## Screencast
CleanShot 2022-04-09 at 23 05 54
CleanShot 2022-04-09 at 23 06 43

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by Aayush9029** [comment]
#1352  seems to have been squashed here, will submit a new pr

**Comment by Aayush9029** [comment]
Should work now

File: extensions/itch-io/package.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    "author": "Aayush9029",
+    "categories": [
+        "Fun",
+        "Other"
+    ],
+    "license": "MIT",
+    "commands": [{
+        "name": "index",
+        "title": "Browse Games",
+        "subtitle": "Browse Popular Games",
```
Comment (mattisssa) [inline extensions/itch-io/package.json:16]:
```suggestion
        "subtitle": "itch.io",
```

File: extensions/itch-io/package.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    "icon": "command-icon.png",
+    "author": "Aayush9029",
+    "categories": [
+        "Fun",
+        "Other"
+    ],
+    "license": "MIT",
+    "commands": [{
+        "name": "index",
+        "title": "Browse Games",
```
Comment (mattisssa) [inline extensions/itch-io/package.json:15]:
```suggestion
        "title": "Search Games",
```

File: extensions/itch-io/package.json [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        "subtitle": "Browse Popular Games",
+        "description": "Explore current most popular game in itch.io",
```
Comment (mattisssa) [inline extensions/itch-io/package.json:17]:
```suggestion
        "description": "Explore the most popular games in itch.io",
```

File: extensions/itch-io/src/components/categoryDropdown.tsx [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    <List.Dropdown
+      tooltip="Select Category"
+      storeValue={true}
+      onChange={(categoryValue) => {
+        onCategoryTypeChange(categoryValue);
+      }}
+    >
+      <List.Dropdown.Section title="Game Categories">
+        {categories.map((category: string) => (
+          <List.Dropdown.Item key={category + ""} title={category + ""} value={"/tag-" + category} />
```
Comment (mattisssa) [inline extensions/itch-io/src/components/categoryDropdown.tsx:18]: Why do you need to add empty strings for `category + ""` ?

**Comment by Aayush9029** [comment]
@mattisssa Lmk if there's anything else that needs to be fixed / changed!

**Review by mattisssa — APPROVED** [review]

**Pull request merged**